_includes/footer.html

```diff
@@ -20,53 +20,63 @@
-                        <li><a href="https://www.docker.com/community/docker-captains">Docker Captains</a></li>
+                        <li><a href="https://www.docker.com/community/get-involved/developer-preview">Preview Program</a></li>
                     </ul>
                 </div>
                 <div class="col-xs-12 col-sm-3 col-md-3">
                     <ul class="footer_links">
                         <li><b><a href="https://www.docker.com/products/docker-desktop">Pricing</a></b></li>
-                        <li><a href="https://www.docker.com/pricing/faq">FAQs</a></li>
-                        <li><a href="https://www.docker.com/partners/programs">Docker Verified Publisher Program</a></li>
+                        <li><a href="https://www.docker.com/pricing/faq">FAQ</a></li>
+                        <li><a href="https://www.docker.com/partners/programs">Docker Verified Publisher</a></li>
+                        <li><a href="https://www.docker.com/partners">Partners</a></li>
                     </ul>
                 </div>
                 <div class="col-xs-12 col-sm-3 col-md-3">
                     <ul class="footer_links">
-                        <li><b><a href="https://www.docker.com/company" target="_blank" rel="noopener">Company</a></b></li>
-                        <li><a href="https://www.docker.com/company">About Us</a></li>
-                        <li><a href="https://www.docker.com/blog/" target="_blank" rel="noopener">Blog</a></li>
-                        <li><a href="https://www.docker.com/customers">Customers</a></li>
-                        <li><a href="https://www.docker.com/partners">Partners</a></li>
+                        <li><b><a href="https://www.docker.com/company" target="_blank" rel="noopener">About Us</a></b></li>
+                        <li><a href="https://www.docker.com/what-container">What is a Container?</a></li>
+                        <li><a href="https://www.docker.com/why-docker">Why Docker?</a></li>
+                        <li><a href="https://www.docker.com/events">Virtual Events</a></li>
+                        <li><a href="https://www.docker.com/swag">Swag Store
+                        </a></li>
                         <li><a href="https://www.docker.com/company/newsroom">Newsroom</a></li>
                         <li><a href="https://www.docker.com/careers">Careers</a></li>
                         <li><a href="https://www.docker.com/company/contact">Contact Us</a></li>
+                        <li><a href="https://www.docker.com/customers">Customers</a></li>
+                        <li><a href="https://www.docker.com/newsletter-subscription">Newsletter</a></li>
                     </ul>
                 </div>
             </div>
             <div class="footer-nav">
                 <nav class="footer_sub_nav">
                     <ul class="menu">
+                        <li><a href="https://www.docker.com/legal/docker-terms-service">Terms of Service</a></li>
```

Comment:
Note that we also have `legal` in the list here (which gives an overview, including this one); wondering if there's a bit of an overlap 🤔